src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.4.0.0.cs

```diff
@@ -57,142 +56,158 @@ public static void NonExistentAction_Throws_ActionNotSupportedException()
     public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutException()
     {
         TimeSpan serviceOperationTimeout = TimeSpan.FromMilliseconds(10000);
-        BasicHttpBinding binding = new BasicHttpBinding();
-        binding.SendTimeout = TimeSpan.FromMilliseconds(5000);
-        ChannelFactory<IWcfService> factory = new ChannelFactory<IWcfService>(binding, new EndpointAddress(Endpoints.HttpBaseAddress_Basic));
+        BasicHttpBinding binding = null;
+        EndpointAddress endpointAddress = null;
+        ChannelFactory<IWcfService> factory = null;
+        IWcfService serviceProxy = null;
+        Stopwatch watch = null;
 
-        Stopwatch watch = new Stopwatch();
-        try
+        // *** VALIDATE *** \\
+        TimeoutException exception = Assert.Throws<TimeoutException>(() =>
         {
-            var exception = Assert.Throws<TimeoutException>(() =>
+            // *** SETUP *** \\
+            binding = new BasicHttpBinding();
+            binding.SendTimeout = TimeSpan.FromMilliseconds(5000);
+            endpointAddress = new EndpointAddress(Endpoints.HttpBaseAddress_Basic);
+            factory = new ChannelFactory<IWcfService>(binding, endpointAddress);
+            serviceProxy = factory.CreateChannel();
+            watch = new Stopwatch();
+
+            // *** EXECUTE *** \\
+            try
             {
-                IWcfService proxy = factory.CreateChannel();
+                watch = new Stopwatch();
                 watch.Start();
-                proxy.EchoWithTimeout("Hello", serviceOperationTimeout);
-            });
-        }
-        finally
-        {
-            watch.Stop();
-        }
+                serviceProxy.EchoWithTimeout("Hello", serviceOperationTimeout);
+            }
+            finally
+            {
+                // *** ENSURE CLEANUP *** \\
+                watch.Stop();
+                ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
+            }
+        });
 
+        // *** ADDITIONAL VALIDATION *** \\
         // want to assert that this completed in > 5 s as an upper bound since the SendTimeout is 5 sec
         // (usual case is around 5001-5005 ms) 
         Assert.InRange<long>(watch.ElapsedMilliseconds, 4985, 6000);
```

Comment:
>Assert.InRange [](start = 8, length = 14)

Nit: I might make this an Assert.True so we could have a more descriptive error message to help diagnose a lab failure.  Not blocking.